### DISCUSSION TITLE
Resolve multiple domain names to one Wordpress site

### DISCUSSION THREAD

**Discussion by woobenjamin** [EasyEngine/easyengine#1638] [Support Requests] [answered]
Hello all,

I've done a lot of Googling, and what I've found doesn't seem to apply to the latest version of EasyEngine. 

What I'm trying to do is to resolve multiple domain names to one Wordpress site. Here's an example ..

### Domains I own
example.com
example.co.uk
example.com.hk

### What I'm trying to do
I'd like **example co.uk** and **example.com.hk** to resolve to my site **example.com**
In the future, if I do something like **example.de** for it to resolve to de.example.com so I can have multi-language support.
I'd like to be able to have LetsEncrypt for example.co.uk, example.com.hk, etc. etc. as well.

I'm guessing it might be an NGINX configuration, but I've never used NGINX before, or maybe it's not an NGINX thing. 

Could someone point me in the right direction? Whether it's another URL, or if you I need to edit something specific to ee. 

Thank you in advance.

/b

**Comment by dhsathiya** [comment, marked as answer]
You can use [**alias domain**](https://easyengine.io/commands/site/update/) feature of EasyEngine.

What you have to do is, 
1. create a site - example.com 
2. add alias domain - ` ee site update example.com --add-alias-domains=example.co.uk,example.com.hk`

**Comment by woobenjamin** [comment]
Thank you.. that was REALLY easy. I guess i need a deeper dive into the docs. Thanks again 


--- 

Ben Woo | +1 (212) 724-1188 

***@***.*** 


On Nov 23, 2021 14:20, Devarshi Sathiya ***@***.***> wrote: 

You can use alias domain feature of EasyEngine.

What you have to do is,

create a site - example.com

add alias domain - ee site update example.com --add-alias-domains=example.co.uk,example.com.hk

**Comment by Alicecommu** [comment]
- Upload the plugin files to the /wp-content/plugins/multiple-domain directory, or install the plugin through the. [WordPress Plugins](https://klasikthemes.com/wordpress-amp-plugins) screen directly.
- Activate the plugin through the 'Plugins' screen in WordPress.
- Use the Settings -> General screen to configure your additional domains.
